### PR TITLE
small fixes

### PR DESCRIPTION
**Pull request by gsajko**
added dependencies, and place to provide open ai keys

### CODE REVIEW

File: notebooks/youtube_transcript_search.ipynb [hunk 1]
```diff
@@ -28,7 +28,7 @@
     }
    ],
    "source": [
-    "pip install --quiet openai tqdm ratelimiter retry"
+    "pip install --quiet openai tqdm ratelimiter retry datasets lance"
```
Comment (changhiskhan) [inline notebooks/youtube_transcript_search.ipynb:31]: the pypi package is named `pylance` (lance was already taken unfortunately).

**Review by changhiskhan — COMMENTED** [review]
Thanks for the PR. I'm happy to share the dataset btw

**Comment by gsajko** [comment]
are they big? I was thinking about including it into repo, so other people trying tutorial wouldn't need to use their openai credits. I think it costs around ~15$ to create embeddings. Cheers

**Comment by changhiskhan** [comment]
here's the link https://eto-public.s3.us-west-2.amazonaws.com/chatbot.tar.gz

File: notebooks/youtube_transcript_search.ipynb (hunk 1 above)
Comment (gsajko) [reply notebooks/youtube_transcript_search.ipynb]: ok, it's corrected to pylance

**Review by changhiskhan — APPROVED** [review]
thanks!

**Pull request merged**